packages/docs/modules/ROOT/pages/public-deploy.adoc

```diff
@@ -1,7 +1,7 @@
 [[deploy-your-contracts-to-a-public-network]]
 = Deploy your contracts to a public network
 
-This tutorial link:first[builds on the first tutorial] where we created a new OpenZeppelin SDK project, and created a simple `Counter` contract in a local development network. We will now see how to deploy this contract to a public network.  
+This guide link:first[builds on the first tutorial] where we created a new OpenZeppelin SDK project, and created a simple `Counter` contract in a local development network. We will now see how to deploy this contract to a public network.  
```

Comment:
For internal links we should use Antora crossreferences, as they are more reliable than relative links.

```suggestion
This guide xref:first.adoc[builds on the first tutorial] where we created a new OpenZeppelin SDK project, and created a simple `Counter` contract in a local development network. We will now see how to deploy this contract to a public network.  
```